src/main/java/uk/gov/hmcts/divorce/document/content/ConditionalOrderCommonContent.java

```diff
@@ -64,6 +77,44 @@ public List<RefusalReason> generateLegalAdvisorComments(ConditionalOrder conditi
         }
     }
 
+    public DocumentType getCoverLetterDocumentType(
+        final CaseData caseData,
+        final Applicant applicant,
+        final boolean isClarificationRefusal
+    ) {
+        if (caseData.getIsJudicialSeparation().toBoolean()) {
```

Comment:
This might throw NPE if `isJudicialSeparation` is null? When it comes to merging in master, may be replace it with `caseData.isJudicialSeparationCase()` which is null safe

Reply:
Yep.  Refactored.